js/components/ExampleChart.vue

```diff
@@ -18,7 +23,8 @@ export default {
     return {
       messages: {
         captionIntro: 'Example',
-        captionTitle:  ': This charted data has a screen date, is true for screened in = yes, and has a Trend Line comparing collected data to a target count',
+        captionTitle:
+          ': This charted data has a screen date, is true for screened in = yes, and has a Trend Line comparing collected data to a target count',
```

Comment:
Breaks like this bother me, but probably nothing we can do. Reminds me of what STS does to params in javadocs.